test-integration/repay-loan/2_repay_loan_eth_4_payments.js

```diff
@@ -0,0 +1,232 @@
+// Util classes
```

Comment:
As this integration tests refers to LINK token, I think it'd be better if we also mention that in the filename as follows:
2_repay_loan_link_4_payments.js

Reply:
Good point, changed